src/tests/ftest/util/ior_test_base.py

```diff
@@ -92,28 +97,26 @@ def create_pool(self):
 
     def create_cont(self):
         """Create a TestContainer object to be used to create container."""
-        # TO-DO: Enable container using TestContainer object,
-        # once DAOS-3355 is resolved.
+        # Enable container using TestContainer object,
         # Get Container params
-        #self.container = TestContainer(self.pool)
-        #self.container.get_params(self)
-
+        self.container = TestContainer(self.pool)
+        self.container.get_params(self)
         # create container
-        # self.container.create()
-        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
+        self.container.create(con_in=self.co_prop)
+        # env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
         # command to create container of posix type
-        cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
-            self.ior_cmd.daos_pool.value, self.ior_cmd.daos_svcl.value)
-        try:
-            container = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                         shell=True)
-            (output, err) = container.communicate()
-            self.log.info("Container created with UUID %s", output.split()[3])
-
-        except subprocess.CalledProcessError as err:
-            self.fail("Container create failed:{}".format(err))
-
-        return output.split()[3]
+        # cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
+        #   self.ior_cmd.daos_pool.value, self.ior_cmd.daos_svcl.value)
+        # try:
+        #    container = subprocess.Popen(cmd, stdout=subprocess.PIPE,
+        #                                 shell=True)
+        #    (output, err) = container.communicate()
+        #    self.log.info("Container created with UUID %s", output.split()[3])
+        #
+        # except subprocess.CalledProcessError as err:
+        #   self.fail("Container create failed:{}".format(err))
+        #
+        # return output.split()[3]
```

Comment:
All the lines from 106-119 which are commented out now should be removed if not needed anymore.

Reply:
Done.